docs/products/mysql/concepts/mysql-memory-usage.rst

```diff
@@ -0,0 +1,16 @@
+MySQL memory usage
+==================
+
+MySQL InnoDB buffer pools are **pre-allocated** and shown as **used memory** even without user activity or stored data in the database. Because of this pre-allocated memory, it's expected high memory usage within all Aiven MySQL plans. 
+
+.. important::
+    
+    The high memory usage on MySQL Aiven services does not indicate any particular issue. 
+
+InnoDB buffer pools are set to values between 15% to 80% of total available memory. Smaller plans use a smaller percentage of total memory for buffer pools because the operating system and management infrastructure components and other MySQL features may consume a considerable portion of the available memory. A sufficient amount of free memory is reserved to run extra utilities while backing up the database.
+
+The MySQL documentation references buffer pool sizing explicitly:
```

Comment:
Do we have a reference for this? I would put also the link to the documentation stating the fact